### PR TITLE
Updated functionality for RDP and external tools.

### PR DESCRIPTION
**Pull request by dotEsuS**

### CODE REVIEW

File: 16.04/usr.bin.remmina [hunk 1]
```diff
@@ -1,9 +1,3 @@
-# Author: Simon Deziel
-# vim:syntax=apparmor
-# TODO: support more than VNC client
-
-#include <tunables/global>
```
Comment (simondeziel) [inline 16.04/usr.bin.remmina]: That include is required (at least for @{HOME}).

File: 16.04/usr.bin.remmina [hunk 2]
```diff
@@ -53,6 +50,10 @@
        member=Set,
 
   /usr/share/glib-2.0/schemas/gschemas.compiled r,
+  /home/*/.config/freerdp/known_hosts rw,
```
Comment (simondeziel) [inline 16.04/usr.bin.remmina]: /home/*/ should almost always be replaced by "owner @{HOME}".

File: 16.04/usr.bin.remmina [hunk 3]
```diff
@@ -53,6 +50,10 @@
...
+  /usr/lib/x86_64-linux-gnu/freerdp/rdpsnd-client-alsa.so mr,
+  /usr/lib/x86_64-linux-gnu/freerdp/rdpsnd-client-pulse.so mr,
+  /lib/x86_64-linux-gnu/ld-*.so mr,
```
Comment (simondeziel) [inline 16.04/usr.bin.remmina]: I'm curious, the remmina package doesn't recommend/suggest freerdp so does it really try to load some of it's plugins/.so?

File: 16.04/usr.bin.remmina [hunk 4]
```diff
@@ -66,6 +67,15 @@
   # plugins
   /usr/lib/@{multiarch}/remmina/plugins/*.so rm,
 
-  # Site-specific additions and overrides. See local/README for details.
-  #include <local/usr.bin.remmina>
+  #external tools
+  /usr/share/remmina/external_tools/ r,
+  /usr/share/remmina/external_tools/launcher.sh r,
+  /usr/share/remmina/external_tools/launcher.sh rix,
```
Comment (simondeziel) [inline 16.04/usr.bin.remmina]: Which rule is the right one for launcher.sh?

File: 16.04/usr.bin.remmina [hunk 5]
```diff
@@ -66,6 +67,15 @@
...
+  /usr/bin/gnome-terminal rix,
+  /usr/bin/gnome-terminal.real mr,
+  /usr/bin/gnome-terminal.real rix,
```
Comment (simondeziel) [inline 16.04/usr.bin.remmina]: Which rule is the right one for gnome-terminal.real?

File: 16.04/usr.bin.remmina [hunk 6]
```diff
@@ -66,6 +67,15 @@
...
+
+  /usr/bin/python3.5 ix,
+  /bin/dash mr,
+  /bin/dash rix,
```
Comment (simondeziel) [inline 16.04/usr.bin.remmina]: Same for dash.

File: 16.04/usr.bin.remmina (hunk 3 above)
Comment (simondeziel) [reply 16.04/usr.bin.remmina]: Is this related to running gnome-terminal contained by that profile (ix)?

Comment (dotEsuS) [reply 16.04/usr.bin.remmina]: I'm not sure what is going on/wrong here... I seem to be able to use RDP without the extra lines.

**Comment by dotEsuS** [comment]
The extra tools (installed by default) are ping.sh ; nslookup.sh and traceroute.sh .
I have not tested the 2 filezilla tools.

File: 16.04/usr.bin.remmina (hunk 3 above)
Comment (dotEsuS) [reply 16.04/usr.bin.remmina]: `owner @{HOME}/.config/freerdp/known_hosts rw,` seems to be necessary for my remmina profiles.

**Comment by simondeziel** [comment]
Sorry for the delay in response. If you come up with a cleaned pull request, I'll look into it. Thanks.

**Pull request closed without merging**